MerlinAU.sh

```diff
@@ -706,21 +708,26 @@ else
 fi
 
 ## Recommended 15 minutes BEFORE the F/W Update ##
-readonly ScriptAU_CRON_DefaultSchedule="45 11 * * *"
+readonly ScriptAU_CRON_DefaultSchedule="45 23 * * *"
+
+## For Automatic Script Updates Cron Schedule ##
+readonly SW_Update_CRON_DefaultSchedDays="* x *"
 
 readonly CRON_MINS_RegEx="([0-9]|[1-5][0-9])"
 readonly CRON_HOUR_RegEx="([0-9]|1[0-9]|2[0-3])"
 
 readonly CRON_DAYofMONTH_rexp1="([1-9]|[1-2][0-9]|3[0-1])"
 readonly CRON_DAYofMONTH_rexp2="${CRON_DAYofMONTH_rexp1}[-]${CRON_DAYofMONTH_rexp1}"
-readonly CRON_DAYofMONTH_rexp3="${CRON_DAYofMONTH_rexp1}([,]${CRON_DAYofMONTH_rexp1})+"
-readonly CRON_DAYofMONTH_RegEx="($CRON_DAYofMONTH_rexp1|$CRON_DAYofMONTH_rexp2|$CRON_DAYofMONTH_rexp3)"
+readonly CRON_DAYofMONTH_rexp3="${CRON_DAYofMONTH_rexp2}[/][2-9]"
+readonly CRON_DAYofMONTH_rexp4="${CRON_DAYofMONTH_rexp1}([,]${CRON_DAYofMONTH_rexp1})+"
+readonly CRON_DAYofMONTH_RegEx="($CRON_DAYofMONTH_rexp1|$CRON_DAYofMONTH_rexp2|$CRON_DAYofMONTH_rexp3|$CRON_DAYofMONTH_rexp4)"
 
-readonly CRON_DAYofWEEK_NAME="([S|s]un|[M|m]on|[T|t]ue|[W|w]ed|[T|t]hu|[F|f]ri|[S|s]at)"
-readonly CRON_DAYofWEEK_rexp1="${CRON_DAYofWEEK_NAME}|[0-6]"
-readonly CRON_DAYofWEEK_rexp2="${CRON_DAYofWEEK_NAME}[-]${CRON_DAYofWEEK_NAME}|[0-6][-][0-6]"
-readonly CRON_DAYofWEEK_rexp3="${CRON_DAYofWEEK_NAME}([,]${CRON_DAYofWEEK_NAME})+|[0-6]([,][0-6])+"
-readonly CRON_DAYofWEEK_RegEx="($CRON_DAYofWEEK_rexp1|$CRON_DAYofWEEK_rexp2|$CRON_DAYofWEEK_rexp3)"
+readonly CRON_DAYofWEEK_Names="([S|s]un|[M|m]on|[T|t]ue|[W|w]ed|[T|t]hu|[F|f]ri|[S|s]at)"
```

Comment:
I just realized I left all my comments pending on this review again. Whoops my bad... Submitting review now!

Approved regex changes